config/manager/manager.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
       - name: cluster-logging-operator
         image: quay.io/openshift-logging/cluster-logging-operator:latest
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
This is probably not intended to go to production right?

Reply:
This just got pushed with my development and testing by mistake. I will remove it soon.